pkgs/by-name/te/tenv/package.nix

```diff
@@ -38,9 +38,9 @@ buildGoModule rec {
 
   meta = {
     changelog = "https://github.com/tofuutils/tenv/releases/tag/v${version}";
-    description = "A version manager for OpenTofu, Terraform and Terragrunt written in Go";
+    description = "OpenTofu, Terraform, Terragrunt and Atmos version manager, written in Go.";
     homepage = "https://github.com/tofuutils/tenv";
     license = lib.licenses.asl20;
-    maintainers = with lib.maintainers; [ rmgpinto ];
+    maintainers = with lib.maintainers; [ rmgpinto nmishin kvendingoldo ];
```

Comment:
Why do we need to add two people here at once?

Reply:
because we're owners of tenv and it will be easier to support it together

Reply:
Okay then